src/Epicea/EpMonitor.class.st

```diff
@@ -528,7 +535,9 @@ EpMonitor >> subscribeToSystemAnnouncer [
 		(ClassRepackaged -> [ :ann | self classRepackaged: ann ]).
 		(ClassRenamed -> [ :ann | self classRenamed: ann ]).
 		(ClassCommented -> [ :ann | self classCommented: ann ]).
-		(MethodRecategorized -> [ :ann | self methodRecategorized: ann ]) } asDictionary keysAndValuesDo: [ :announcement :block |
+		(MethodRecategorized -> [ :ann | self methodRecategorized: ann ]) .
+		(SettingUpdated -> [ : ann | self settingUpdated: ann ])
```

Comment:
I think that #SettingUpdated should be sent on the codeSupportAnnouncer instead of the codeChangeAnnouncer.

We can discuss what is the difference together if you want